src/uct/tcp/tcp_iface.c

```diff
@@ -766,7 +772,10 @@ ucs_status_t uct_tcp_query_devices(uct_md_h md,
             continue;
         }
 
-        if (!ucs_netif_is_active(entry->d_name)) {
+        uct_md_iface_config_read(md, "tcp", NULL, NULL, &config);
+        iface_config = ucs_derived_of(config, uct_tcp_iface_config_t);
```

Comment:
we can add configuration for tcp memory domain and put it there (so it will be available before creating the interface)